tests/unit/model/model-range-test.ts

```diff
@@ -367,4 +369,34 @@ module('Unit | model | model-range', () => {
       });
     });
   });
+  module('Unit | model | model-range | compare', () => {
```

Comment:
more tests on the range comparison might be warranted

Reply:
yeah, good point
I'll evaluate if its still needed, I don't think I ended up using it much so I might just get rid of it